app/browser/api/ledger.js

```diff
@@ -1117,6 +1119,38 @@ const pageDataChanged = (state, viewData = {}, keepInfo = false) => {
   return state
 }
 
+const checkBrowserActivityTime = (state) => {
+  // Fuzzing threshold
+  const minTime = 30 * ledgerUtil.milliseconds.minute
+  const publishers = ledgerState.getPublishers(state)
+
+  if (publishers.isEmpty()) {
+    return state
+  }
+
+  const publisherKeys = Array.from(publishers.keys())
+  const ledgerStatus = ledgerState.getAboutProp(state, 'status')
+  const curBrowsingTime = ledgerState.getAboutProp(state, 'browsingTime') || 0
+
+  // Check cached browsing time to avoid unneeded recalculation
+  if (curBrowsingTime >= minTime && ledgerStatus !== ledgerStatuses.FUZZING) {
```

Comment:
We should do browser activity check only if `ledgerStatuses` is `FUZZING`

`if (curBrowsingTime >= minTime || ledgerStatus !== ledgerStatuses.FUZZING) {`

Reply:
In the current conditional, the activity calculation is not done and state is returned as `curBrowsingTime >= minTime && ledgerStatus !== ledgerStatuses.FUZZING` are the requirements. So it will proceed with the check if `ledgerStatus !== ledgerStatuses.FUZZING` or am I missing something?